### PR TITLE
Update postinstall notes for Spark and Chronos

### PR DESCRIPTION
**Pull request by elingg**

### CODE REVIEW

**Comment by elingg** [comment]
@jsancio or @mgummelt, please take a look. Thanks!

File: repo/packages/S/spark/0/package.json [hunk 1]
```diff
@@ -11,5 +11,5 @@
     "icon-medium": "https://downloads.mesosphere.io/spark/assets/icon-service-spark-medium.png",
     "icon-large": "https://downloads.mesosphere.io/spark/assets/icon-service-spark-large.png"
   },
-  "postInstallNotes": "Spark cluster mode on Mesos is ready!"
+  "postInstallNotes": "Spark DCOS Service has been successfully installed!\nNote that the service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.\nIn order for Spark to start successfully, all resources must be available in the cluster, including ports, CPU shares, and RAM.\nWe recommend a minimum of one node with at least 1 CPU and 2GB of RAM available for use by the Spark Service.\n\n\tDocumentation: https://spark.apache.org/docs/latest/running-on-mesos.html\n\tIssues: https://issues.apache.org/jira/browse/SPARK"
```
Comment (mgummelt) [inline repo/packages/S/spark/0/package.json]: I'm not sure what "all resources must be available in the cluster" means

**Comment by mgummelt** [comment]
LGTM

File: repo/packages/S/spark/0/package.json (hunk 1 above)
Comment (elingg) [inline repo/packages/S/spark/0/package.json]: Good point, that line is more relevant for HDFS and Cassandra, which require multiple nodes with resources available for them to start and achieve health. Removing that sentence.

**Pull request merged**